dd-sdk-android/src/main/kotlin/com/datadog/android/error/internal/DatadogExceptionHandler.kt

```diff
@@ -75,10 +74,16 @@ internal class DatadogExceptionHandler(
     }
 
     private fun createCrashMessage(throwable: Throwable): String {
-        return if (throwable.message.isNullOrBlank()) {
-            MESSAGE
+        val rawMessage = throwable.message
+        return if (rawMessage.isNullOrBlank()) {
+            val className = throwable.javaClass.canonicalName
```

Comment:
you do not want to use the `javaClass.simpleName` in case the `cannonicalName` is null here ?

Reply:
That's a good idea indeed